packages/auth-server/services/socket-io.js

```diff
@@ -0,0 +1,94 @@
+const Emittery = require('emittery')
+const {getAPISig, newClientDB} = require("./hub-helpers");
+
+const threadDbId = [1, 85, 2, 162, 50, 48, 209, 242, 140, 224, 92, 53, 163, 197, 239, 228, 119, 247, 83, 123, 70, 224,
+  215, 43, 215, 103, 142, 189, 149, 148, 228, 164, 171, 38]
+
+module.exports = (io) => {
+    io.on('connection', (socket) => {
+      console.log('New user connected');
+
+      socket.on('message', async (msg) => {
```

Comment:
Let's replace the socket messages with appropriate names:
Server side:
`message` -> `authInit`
`challenge` -> `challengeResp`

Client-side:
message -> `authMsg`